### ISSUE TITLE
Logging into nexus 1.3.11

### ISSUE THREAD

**Issue by stumbles** [TanninOne/modorganizer#960]
Anytime I click on "Check all for update" a Login failed window pops up saying it failed and do I want to try again. Yes, of course. Then in the bottom window of Mod Organizer it just shows login failed and check your password. 

Of course my password is correct, been verified by logging into nexus and steam via a browser. I have;

1. verified the NMM version workaround (currently 0.61.23)
2. enabled automatically log-in to nexus with correct username/passwd
3. have the correct name/passwd for the steam tab in settings
4. has 72850 as the current steam app id
5. running windows 7 professional
6. yes, have installed MO before and not had this problem

It doesn't have a problem downloading mods but ran nxmhandler.exe anyway and tried deleting the webcache. If I disable automatic login then check all for updates works fine.

Here is the output of the error window;

> 07:06:23 [D] Working directory: D:\steam\steamapps\common\Skyrim\Mod Organizer
07:06:23 [D] MO at: D:\steam\steamapps\common\Skyrim\Mod Organizer
07:06:23 [D] initializing core
07:06:23 [D] initialize plugins
07:06:23 [D] looking for plugins in D:\steam\steamapps\common\Skyrim\Mod Organizer\plugins
07:06:23 [D] loaded plugin "bsaExtractor.dll"
07:06:23 [D] loaded plugin "checkFNIS.dll"
07:06:23 [D] loaded plugin "diagnoseBasic.dll"
07:06:23 [D] loaded plugin "gameFallout3.dll"
07:06:23 [D] loaded plugin "gameFalloutNV.dll"
07:06:23 [D] loaded plugin "gameOblivion.dll"
07:06:23 [D] loaded plugin "gameSkyrim.dll"
07:06:23 [D] loaded plugin "iniEditor.dll"
07:06:23 [D] loaded plugin "installerBAIN.dll"
07:06:23 [D] loaded plugin "installerBundle.dll"
07:06:23 [D] loaded plugin "installerFomod.dll"
07:06:23 [D] loaded plugin "installerManual.dll"
07:06:23 [D] loaded plugin "installerNCC.dll"
07:06:23 [D] loaded plugin "installerQuick.dll"
07:06:23 [D] loaded plugin "NMMImport.dll"
07:06:23 [D] loaded plugin "previewBase.dll"
07:06:23 [D] loaded plugin "pyCfg.py"
07:06:23 [D] loaded plugin "proxyPython.dll"
07:06:23 [D] downloads after refresh: 165
07:06:23 [D] game path: D:\steam\steamapps\common\Skyrim
07:06:23 [D] managing game at D:\steam\steamapps\common\Skyrim
07:06:23 [D] setting up configured executables
07:06:23 [D] configured profile: Skyrim Revisited
07:06:23 [D] initializing tutorials
07:06:23 [D] loaded language en-US
07:06:24 [D] interface Local Area Connection seems to be up (address: fe80::c060:13be:7da7:4053%11)
07:06:24 [D] displaying main window
07:06:26 [D] D:\steam\steamapps\common\Skyrim\Mod Organizer\profiles\Skyrim Revisited\modlist.txt saved
07:06:27 [D] logging anvilton in on Nexus
07:06:29 [D] login failed: Please check your password
07:06:29 [D] login failed: Please check your password
 [] 

Don't know what to do at this point.

**Comment by Gernash** [comment]
firewall blocking you? either MS or 3rd party?

**Comment by stumbles** [comment]
No, none of that and haven't changed anything in any of those areas from when it once did work.

**Comment by Gernash** [comment]
Firewall is seperate to MSE
haev you had a look at your ModOrganizer.ini file there is a parameter nexus_password= you could delete everything after the = and see if entering it again via Modorganizer setting tab in case there is glitch in the matrix otherwise I'd disable the firewall and check that it's not the cause (Not security essentuals)

**Comment by stumbles** [comment]
Erp???

For some reason I had to re-register at nexus, all is well now. Sorry for the noise.
